tests/test_intervention.py

```diff
@@ -816,6 +816,101 @@ def test_eproms_p3p_conditions(self):
         user, ds_p3p = map(db.session.merge, (user, ds_p3p))
         self.assertFalse(ds_p3p.display_for_user(user).access)
 
+    def test_truenth_st_conditions(self):
+        # Test the list of conditions expected for SymptomTracker in truenth
+        ds_sm = INTERVENTION.SELF_MANAGEMENT
```

Comment:
*really* minor, but the `ds_*` pattern belongs to decision_support_{p3p, wisercare}.  Probably just want to use `sm` here.

Reply:
Ah right, that makes sense